### PR TITLE
Issue 470 - mismatch in dropdown vs saved search: API

### PR DESCRIPTION
**Pull request by vichargrave**
Client and backend changes all in place.  Ready for review.

### CODE REVIEW

File: goldstone/core/models.py [hunk 1]
```diff
@@ -2411,9 +2411,13 @@ class SavedSearch(models.Model):
                                     help_text='True if this is system-defined')
 
     hidden = models.BooleanField(blank=True, default=False,
-                                 help_text='True if this search should not be'
+                                 help_text='True if this search should not be '
                                            'presented via the view')
 
+    viewer_enabled = models.BooleanField(blank=True, default=True,
+                                         help_text='True if this search should be '
+                                         'presented via the view')
```
Comment (lexjacobs) [inline goldstone/core/models.py]: please disambiguate client from server views.

**Comment by lexjacobs** [comment]
👍

**Comment by vichargrave** [comment]
That's a wrap.  Merging and closing issue-470 - https://github.com/Solinea/goldstone-server/issues/470.

**Comment by vichargrave** [comment]
That's a wrap.  Merging and closing issue-470 (https://github.com/Solinea/goldstone-server/issues/470)

**Pull request closed without merging**